src/model/graph.ts

```diff
@@ -88,6 +88,12 @@ export class Graph implements GraphInterface {
   get analysts() {
     return this._analysts;
   }
+  get libraryVersion() {
```

Comment:
You can just remove the `protected` keyword in lines 50 and 51, then we do not need a getter here.